swarm/swarm.go

```diff
@@ -228,7 +225,6 @@ func NewSwarm(ctx *node.ServiceContext, backend chequebook.Backend, config *api.
 	self.bzz = network.NewBzz(bzzconfig, to, stateStore, stream.Spec, self.streamer.Run)
 
 	// Pss = postal service over swarm (devp2p over bzz)
-	config.Pss = config.Pss.WithPrivateKey(self.privateKey)
```

Comment:
I think that config should bot be mutated in constructor, so I moved this line to api.Config.Init function. @nolash is that ok?

Reply:
fine by me :)